samples/Sentry.Samples.Hangfire/README.md

```diff
@@ -0,0 +1,10 @@
+# Sentry for Hangfire
+
+This is a standard ASP.NET Core app with nothing in it but Hangfire.
```

Comment:
Does the Cron stuff take place in Sentry.Io then?

Reply:
Fresh off the oven: https://blog.sentry.io/cron-monitoring-is-now-generally-available/

Reply:
> Hangfire spins up the jobs. The integration we build just pings Sentry with check-ins. Successful or not. If Sentry fails to get check-ins it can alert u (job is dead?)

That's what I figured... I guess I'm wondering what this sample is for. You couldn't use it without deploying it on a public URL somewhere and configuring a cron on Sentry right?